javascript/builtins/Promise.json

```diff
@@ -374,6 +374,59 @@
             }
           }
         },
+        "incumbent_realm_tracking": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise#Incumbent_realm_tracking",
+            "spec_url": "https://html.spec.whatwg.org/#incumbent",
```

Comment:
```suggestion
```

Sorry for not catching this in my initial review, but one last change: I suggest we omit the spec URL for now.

I suggest that because — as far I understand — this feature hasn’t actually landed in the spec yet, right? I mean, the https://github.com/whatwg/html/pull/5722 spec patch hasn’t been merged yet.

And since that spec patch is making changes to the **JavaScript specification host hooks** section (8.1.5), it seems like once the spec patch is actually merged, the relevant spec URL won’t be the https://html.spec.whatwg.org/#incumbent section (8.1.3) — but instead, some anchor within the **JavaScript specification host hooks** section. (Although, looking at the patch, even within that section, it doesn’t seem clear to me at least exactly what anchor in there that we should rightly be linking to…)

Reply:
Added!